01week/datatypes.js

```diff
@@ -0,0 +1,62 @@
+'use strict'
+//1. Write a JavaScript program to display the current day and time.
+const todaysDate = new Date();
+console.log(todaysDate);
+
+//2. Write a JavaScript program to convert a number to a string.
+
+const numberToString = (num1) => {
+console.log(Number(num1))
+}
+numberToString(22)
+
+//3. Write a JavaScript program to convert a string to the number.
+const string = '18';
+const integer = parseInt(string, 10);
```

Comment:
this needs to be a function that takes in any string and converts it to a number